src/styles/index.css

```diff
@@ -0,0 +1,17 @@
+
+@import "./reset.css";
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background: #e5e5e5;
+}
+
+@import "./blocks/card.css";
```

Comment:
Move the import statement for './blocks/card.css' to the top along with other imports.